modules/tensor_mechanics/src/materials/RankTwoCylindricalComponent.C

```diff
@@ -0,0 +1,60 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "RankTwoCylindricalComponent.h"
+#include "RankTwoScalarTools.h"
+
+registerMooseObject("TensorMechanicsApp", RankTwoCylindricalComponent);
+
+InputParameters
+RankTwoCylindricalComponent::validParams()
+{
+  InputParameters params = Material::validParams();
+  params.addClassDescription("Compute a cylindrical stress/strain property of a RankTwoTensor");
+  params.addRequiredParam<MaterialPropertyName>("rank_two_tensor",
+                                                "The rank two material tensor name");
+  params.addParam<std::string>("calculation_name",
```

Comment:
Rename `calculation_name` to `property_name` for consistency with other models. Also, the description should just be something like: 'Name of the material property computed by this model'